src/angular/form-field/form-field/form-field-control.ts

```diff
@@ -0,0 +1,22 @@
+import type { SbbFormFieldElementControl } from '@sbb-esta/lyne-elements/form-field.js';
```

Comment:
you can add documentation, check src/showcase/stories/angular/autocomplete/readme.md for an example. Even if we do it better in the future, I'd recommend to add it now also